tests/kuttl/test-suites/default/watcher/01-assert.yaml

```diff
@@ -114,3 +114,16 @@ commands:
       oc exec -n watcher-kuttl-default openstackclient -- openstack service list -f value -c Name -c Type |[ $(grep -c ^watcher) == 1 ]
```

Comment:
We should also check the values of the three new image parameters in the Watcher object, at around line 7 (you'll need to add spec section.

Reply:
Not sure I have done it correctly. Please have a look at spec changes: https://github.com/raukadah/watcher-operator/blob/webhook_default_images/tests/kuttl/test-suites/default/watcher/01-assert.yaml#L8